### DISCUSSION TITLE
FNIRSI GC-01: Changing the tube

### DISCUSSION THREAD

**Discussion by webrex2000** [Gissio/radpro#152] [Q&A]
I need help as I'm new to this kind of upgrade. I have  a FNIRSI  GC-01 with j613 tube and I want to  use a j321. The pcb has the spots for the clips and I found a tube on aliexpress. Few questions: firmware upgrade must be done before or after  changing the tube? Hobby solder is good for the clips? I understand the voltage for the j321 can be adjusted with new firmware using HV am 'i right? Seems i do not need to remove or add any resistor. Thank you for your help and sorry for my English.

**Comment by alfmck** [comment]
I did that for me. If you made firmware updates to Radpro, initially HV will be 700-850V, which is too high for J321, but not critical. So, you have to come to settings, Geiger tube, HV profile, custom profile, OK, and set PWM frequency let say to 5 kHz and PWM duty cycle to 1.5-3.0%. After that you can change the tube.

**Reply by webrex2000** [reply]
Thank you for your help.... Did you use a normal solder for the clips?

**Reply by alfmck** [reply]
Yes, but the temperature has to be about 400 degrees. I had to do clips by myself.

**Reply by webrex2000** [reply]
I found a j321 in aliexpress with clips included..... 400 degrees the solder..... I hope I can do it

**Reply by webrex2000** [reply]
What about the old tube... Just cut the wires and thats' it right?

**Reply by webrex2000** [reply]
Hi i just updated the firmware and changed to the j321 tube, I used custom Hv profile you suggested but I have 91 volts on the tube, is this normal?

**Reply by alfmck** [reply]
To measure HV on the tube you should use 1GOm resistor, subsequently connected to voltmeter. In order to set right HV, simply try to increase duty cycle step by step and observe the average number of counts per minute. It should be 20-24 CPM, depending on the place you live. And you asked about old tube - I suggest leaving it, parallelly connected to J321.

**Reply by Lanchon** [reply]
i wouldn't recommend that. if you leave both tubes, you need a different calibration for the combo, and sensitivity will depend more on the angle of incidence. plus the optimum voltage for each tube will be different.
